File: Framework/CSharp/Framework/Framework/Web/Mvc/SmartXmlResult.cs
/*
 * 项目：Smartkernel.Framework
 * 作者：曹艳白
 * 邮箱：caoyanbai@139.com
 */
using System.Web.Mvc;

namespace Smartkernel.Framework.Web.Mvc
{
    /// <summary>
    /// Xml结果
    /// </summary>
    public class SmartXmlResult : ContentResult
    {
        /// <summary>
        /// 构造函数
        /// </summary>
        public SmartXmlResult()
            : base()
        {
            this.ContentType = SmartMimeType.Xml;
        }
    }
}

File: Framework/Php/framework/framework/web/mvc/smartxmlresult.php
<?php

/*
 * 项目：Smartkernel.Framework
 * 作者：曹艳白
 * 邮箱：caoyanbai@139.com
 */

namespace Smartkernel\Framework\Web\Mvc {

    use Smartkernel\Framework\Web\SmartMimeType;
    use Smartkernel\Framework\Web\Mvc\SmartContentResult;

/*
     * Xml结果
     */

    class SmartXmlResult extends SmartContentResult {
        /*
         * 构造函数
         */

        function __construct() {
            parent::__construct();
            $this->ContentType = SmartMimeType::Xml;
        }

    }

}